include/etiss/CPUCore.h

```diff
@@ -344,7 +344,8 @@ class CPUCore : public VirtualStructSupport, public etiss::ToString
      * @brief returns the list of all plugins.
      */
     inline std::list<std::shared_ptr<Plugin>> const *getPlugins() { return &plugins; };
-
+  float startTime;
```

Comment:
this could be a bit cleaner by defining normal private members with member naming (`startTime_`) and adding a member function `float getSimulationTime() const`.

we should probably also switch to double for this, because there is no cost and clock_t is 32-bits and floats only have 24 significant binary digits.

Reply:
I agree on making the `startTime` member private and adding a getter; also with switching `startTime` and also any other related variables to double.

Another idea would be to handle output of the statistics file within `CPUCore.cpp`, instead of `bare_etiss_processor`. This would allow ETISS to also output statistic information when it is run in a VP context, read without `bare_etiss_processor.cpp`